src/EnergyPlus/Plant/Component.hh

```diff
@@ -158,7 +158,8 @@ namespace DataPlant {
         false, //	"AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam"
         false, //	"ZoneHVAC:CoolingPanel:RadiantConvective:Water"
         false, //	"HeatPump:PlantLoop:EIR:Cooling"
-        false  //	"HeatPump:PlantLoop:EIR:Heating"
+        false, //	"HeatPump:PlantLoop:EIR:Heating"
+        false  //   "DistrictHeating:Steam"
```

Comment:
This array should be parallel with [`PlantEquipmentType`](https://github.com/NREL/EnergyPlus/blob/68508860518b5f36a81d9ddeb28cc661f229b631/src/EnergyPlus/Plant/Enums.hh#L108) enum. It was missing a slot for `PurchSteam` at the end.
https://github.com/NREL/EnergyPlus/blob/68508860518b5f36a81d9ddeb28cc661f229b631/src/EnergyPlus/Plant/Enums.hh#L209-L211